starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+*{
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans';
+}
+
+nav {
+  position: fixed;
```

Comment:
When you use `position: fixed;` or `position: absolute;` you are excepted to manually position the element. Here, because you want the `nav` to stick at the top of the screen, you should add `top: 0;`.

Another problem is that your `nav` may then disappear behind the image below. To make sure it is always visible, you should give a big enough value to the `z-index` CSS property. Here, `z-index: 1;` would work.